tests/src/upgradeability/tests/functions/proxy_owner.rs

```diff
@@ -0,0 +1,27 @@
+mod success {
+
+    use crate::upgradeability::tests::utils::{
+        abi_calls::{proxy_owner, set_proxy_owner},
+        test_helpers::setup,
+        State,
+    };
+
+    #[tokio::test]
+    async fn returns_initialized_owner() {
```

Comment:
Could you add tests for when things aren't initialized?

Reply:
I've left these out as the initialisation function included in the testing contract is not a part of the library and exists only for test setup convenience.